client/gui/CGuiHandler.h

```diff
@@ -48,33 +49,34 @@ class CGuiHandler
 	CGStatusBar * statusbar;
 
 private:
-	typedef std::list<CIntObject*> CIntObjectList;
+	typedef std::list<CIntObject *> CIntObjectList;
 
 	//active GUI elements (listening for events
 	CIntObjectList lclickable,
-				   rclickable,
-				   mclickable,
-				   hoverable,
-				   keyinterested,
-				   motioninterested,
-	               timeinterested,
-	               wheelInterested,
-	               doubleClickInterested,
-	               textInterested;
+	rclickable,

```

Comment:
I'd say that when declaring multiple names, they should be indented, so it is clear that they belong to the type given in the initial declaration line.